### ISSUE TITLE
Compile Error

### ISSUE THREAD

**Issue by yannhan** [HawkAaron/mxnet-transducer#4]
Hi,

When I tried to compile RNNTLoss with mxnet source code, it showed the error like below. I don't know how to deal with that, could you please help me? I just run "make" in the directory of incubator-mxnet after copying your rnnt loss code in the directory.

Thanks,
![Screenshot from 2019-10-29 14-02-57](https://user-images.githubusercontent.com/43621476/67800235-f10f0100-fa54-11e9-8043-20227a6e226b.png)

**Comment by vlavla** [comment]
https://github.com/HawkAaron/mxnet-transducer/pull/5

**Comment by yannhan** [comment]
Thanks for your reply, but I still get below error. Any idea about this?

![Screenshot from 2019-10-30 16-01-12](https://user-images.githubusercontent.com/43621476/67898583-90ec8d80-fb2e-11e9-8590-47f235a08d32.png)

**Comment by vlavla** [comment]
Have you installed libopenblas-dev?

**Comment by yannhan** [comment]
Yes, I have installed the newest version (0.2.20+ds-4).

**Comment by HawkAaron** [comment]
I just merged a pull request, please update the code and remake.

**Comment by yannhan** [comment]
I changed another workstation to compile, it succeeded. But only CPU tests passed.

![b5ffe6d709ce80f9f0ffb26e91c52a6](https://user-images.githubusercontent.com/43621476/67919259-b5b42580-fb6d-11e9-80fd-4dc39c10fe9d.png)

**Comment by yannhan** [comment]
Just to claim it more clearly.

I followed the step of Manual MXNet Installation of the link https://mxnet.incubator.apache.org/get_started/ubuntu_setup.html#install-mxnet-for-python

From step 1 to Install MXNet for Python.

And it actually successfully installed mxnet.
![aac97aa68489497c4f11c1aca2622fa](https://user-images.githubusercontent.com/43621476/67920470-5dcbed80-fb72-11e9-97b5-162ad2fdede2.png)

But when I tried the test py file. It said no module named 'mxnet'...

![7dccde0aea76223a889e203b0bb18ad](https://user-images.githubusercontent.com/43621476/67920506-818f3380-fb72-11e9-9740-2ce4d592a717.png)
